default.json

```diff
@@ -103,6 +103,15 @@
       "datasourceTemplate": "docker",
       "versioningTemplate": "docker"
     },
+    {
+      "description": "Update Github Action references in the README.md on renovatebot/github-action repository",
+      "fileMatch": ["README.md"],
```

Comment:
I'm not sure how to make it so that `fileMatch` only matches `README.md` and nothing else. Probably needs some escape characters?

Reply:
```suggestion
      "fileMatch": ["^README\\.md$"],
```

It's a regex 😉